contracts/staking/contracts/src/stake/MixinStakeBalances.sol

```diff
@@ -29,135 +29,65 @@ contract MixinStakeBalances is
 {
     using LibSafeMath for uint256;
 
-    /// @dev Returns the total active stake across the entire staking system.
-    /// @return Global active stake.
-    function getGlobalActiveStake()
+    /// @dev Gets global stake for a given status.
+    /// @param stakeStatus INACTIVE or DELEGATED
+    /// @return Global stake for given status.
+    function getGlobalStakeByStatus(IStructs.StakeStatus stakeStatus)
         external
         view
-        returns (IStructs.StakeBalance memory balance)
+        returns (IStructs.StoredBalance memory balance)
     {
-        IStructs.StoredBalance memory storedBalance = _loadSyncedBalance(
-            globalStakeByStatus[uint8(IStructs.StakeStatus.ACTIVE)]
-        );
-        return IStructs.StakeBalance({
-            currentEpochBalance: storedBalance.currentEpochBalance,
-            nextEpochBalance: storedBalance.nextEpochBalance
-        });
-    }
-
-    /// @dev Returns the total inactive stake across the entire staking system.
-    /// @return Global inactive stake.
-    function getGlobalInactiveStake()
-        external
-        view
-        returns (IStructs.StakeBalance memory balance)
-    {
-        IStructs.StoredBalance memory storedBalance = _loadSyncedBalance(
-            globalStakeByStatus[uint8(IStructs.StakeStatus.INACTIVE)]
-        );
-        return IStructs.StakeBalance({
-            currentEpochBalance: storedBalance.currentEpochBalance,
-            nextEpochBalance: storedBalance.nextEpochBalance
-        });
+        balance = _loadSyncedBalance(_globalDelegatedStake);
```

Comment:
Could reduce number of `sload` ops here by moving this into a separate if block.

Reply:
`loadSyncedBalance` copies to memory, so doesn't seem like a huge deal. This function is also never called internally.